dev/Controller/MockController.php

```diff
@@ -0,0 +1,152 @@
+<?php
```

Comment:
It might not be in our DoD, but I strongly feel all code we write should be strictly typed. Consider adding the declare strict types declaration. And update the method signatures to include typed parameters and return types.

Reply:
I agree, only adding the strict type declaration seemed to be sufficient no other code changes were needed.